lib/render/GeoImage.cpp

```diff
@@ -139,7 +139,7 @@ int GeoImage::TiffReadImage(int dirnum, unsigned char *texture) const
 {
     VAssert(_tif != NULL);
 
-    uint32 *texuint32 = (uint32 *)texture;
+    uint32_t *texuint32_t = (uint32_t *)texture;
```

Comment:
@sgpearse  can you do a sanity check to not change variable names? More specifically, this PR only changes type names from `uint32` to `uint32_t`, but not to change variable names.